ir/function.h

```diff
@@ -157,6 +157,7 @@ class Function final {
                          const std::vector<std::string_view> &src_glbs);
 
   void print(std::ostream &os, bool print_header = true) const;
+  void unroll();
```

Comment:
this shouldn't be placed in the middle of the 2 printing methods. also, it should receive k as argument.